src/window/advisor/trade.c

```diff
@@ -43,6 +47,7 @@ static generic_button resource_buttons[] = {
     {64, 261, 480, RESOURCE_ROW_HEIGHT - 2, button_resource, button_none, 5, 0},
     {64, 302, 480, RESOURCE_ROW_HEIGHT - 2, button_resource, button_none, 6, 0},
     {64, 343, 480, RESOURCE_ROW_HEIGHT - 2, button_resource, button_none, 7, 0},
+    {580, 390, 40, 30, button_policy, button_none, 1, 0}
```

Comment:
Please add this to the third line instead of the bottom and change all the checks accordingly (the resources should start at index 2 before and now they must start at index 3).

Also, don't forget to do it to `get_tooltip` as well (right now the "click here for industry status" tooltip shows for the caravanserai button when you hover over its position, even if the button is not showing because there is no caravanserai)